src/DynamicRupture/Initializers/ImposedSlipRatesInitializer.cpp

```diff
@@ -85,10 +89,10 @@ void ImposedSlipRatesInitializer::rotateSlipToFaultCS(
     real scalarProduct = MeshTools::dot(crossProduct, fault.normal);
     real sin = std::sqrt(1 - cos * cos) * std::copysign(1.0, scalarProduct);
     for (size_t pointIndex = 0; pointIndex < misc::numPaddedPoints; ++pointIndex) {
-      real strikeSlip = slip1[ltsFace][pointIndex];
-      real dipSlip = slip2[ltsFace][pointIndex];
-      slip1[ltsFace][pointIndex] = cos * strikeSlip + sin * dipSlip;
-      slip2[ltsFace][pointIndex] = -sin * strikeSlip + cos * dipSlip;
+      imposedSlipDirections[ltsFace][pointIndex][0] =
+          cos * strikeSlip[ltsFace][pointIndex] + sin * dipSlip[ltsFace][pointIndex];
+      imposedSlipDirections[ltsFace][pointIndex][1] =
+          -sin * strikeSlip[ltsFace][pointIndex] + cos * dipSlip[ltsFace][pointIndex];
```

Comment:
the same comments about vectorization. I guess the compiler will find a way to vectorize this code and it will be easy in case of a unit-stride access with respect to `imposedSlipDirections`